packages/next/next-server/lib/router/utils/route-regex.ts

```diff
@@ -25,14 +25,16 @@ export function getRouteRegex(
         $1 = $1.slice(2, -2)
       }
       const isCatchAll = /^(\\\.){3}/.test($1)
+      if (isCatchAll) {
+        $1 = $1.slice(6)
```

Comment:
@Timer This file isn't loaded in the browser, right? I think there's some more regexes here that can be optimized by  using string functions. But it could slightly increase bundle size or be incompatible with older browsers

Reply:
This code **is** loaded for the browser.